AAI/AAIConnectProfile.md

```diff
@@ -402,7 +403,9 @@ the Broker.
 1.  Claim Clearinghouses MUST trust at least one Broker.
 
     1.  Claim Clearinghouses MAY trust more than one Broker
-
+    
+    2.  The burden of risk assessment is on the Claim Clearinghouse to trust an access token from a broker. RECOMENDED that only a small number of Brokers are trusted by any one Clearinghouse in order to lessen this burden. 
```

Comment:
typo missing double-m: recommended

Reply:
also but not very important, generally we are using uppercase "b" in Broker as a defined term inside this spec.